core/src/test/resources/flows/sleep-task.yml

```diff
@@ -0,0 +1,7 @@
+id: sleep-task-flow
+namespace: io.kestra.tests
+
+tasks:
+  - id: sleep
+    type: io.kestra.core.tasks.scripts.Bash
+    script: sleep 30
```

Comment:
Can you remove this ? Never use

Reply:
done @Skraye

Reply:
Delete the file, not just remove the content please

Reply:
hello @Skraye 
Updated the changes according to your review, Thanks alot for the review , if you have time to review it once more , please check and tell if there is need of any more modifications.